pulser-core/pulser/backend/remote.py

```diff
@@ -122,6 +128,9 @@ def fetch_available_devices(self) -> dict[str, Device]:
             "remote connection."
         )
 
+    def _close_batch(self, batch_id: str) -> SubmissionStatus | None:
+        pass
```

Comment:
- You still have to fill this in on the `PasqalCloud` class. 
- Since this is not an `abstractmethod`, let's raise a `NotImplementedError` here.
- Why are you returning `SubmissionStatus`? I think returning `None` should be fine